File: XAM03012018/RiderJAVA/src/main/java/projectrider/entity/Project.java
package projectrider.entity;

import javax.persistence.*;

@Entity
@Table(name = "projets")
public class Project {

    private Integer id;

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    private String title;

    @Column(nullable = false)
    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    private String description;

    @Column(nullable = false)
    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    private Long budget;

    @Column(nullable = false)
    public Long getBudget() {
        return budget;
    }

    public void setBudget(Long budget) {
        this.budget = budget;
    }

    public Project(String title, String description, Long budget) {
        this.title = title;
        this.description = description;
        this.budget = budget;
    }

    public Project() {    }
}

File: XAM03012018/RiderPHP/src/AppBundle/Entity/Project.php
<?php

namespace AppBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * @ORM\Table(name="projects")
 * @ORM\Entity(repositoryClass="AppBundle\Repository\ProjectRepository")
 */
class Project
{
	/**
	 * @var int
	 * @ORM\Column(name="id", type="integer")
	 * @ORM\Id
	 * @ORM\GeneratedValue(strategy="AUTO")
	 */
	private $id;

	/**
	 * @return int
	 */
	public function getId()
	{
		return $this->id;
	}

	/**
	 * @param int $id
	 */
	public function setId(int $id)
	{
		$this->id = $id;
	}

	/**
	 * @var string
	 * @Assert\NotBlank()
	 * @Assert\Length(min = 2, max = 255,
	 *     minMessage = "Title must be at least {{ limit }} characters long",
	 *     maxMessage = "Title too long! No more than {{ limit }} characters allowed!")
	 * @ORM\Column(name="title", type="string", length=255)
	 */
	private $title;

	/**
	 * @return string
	 */
	public function getTitle()
	{
		return $this->title;
	}

	/**
	 * @param string $title
	 * @return Project
	 */
	public function setTitle($title)
	{
		$this->title = $title;
	}

	/**
	 * @var string
	 * @Assert\NotBlank()
	 * @Assert\Length(min = 2, max = 255,
	 *     minMessage = "Please provide project description!",
	 *     maxMessage = "Description too long! No more than {{ limit }} characters allowed!")
	 * @ORM\Column(name="description", type="string", length=255)
	 */
	private $description;

	/**
	 * @return string
	 */
	public function getDescription()
	{
		return $this->description;
	}

	/**
	 * @param string $description
	 * @return Project
	 */
	public function setDescription($description)
	{
		$this->description = $description;
	}

	/**
	 * @var int
	 * @Assert\NotBlank()
	 * @ORM\Column(name="budget", type="integer")
	 */
	private $budget;

	/**
	 * @return int
	 */
	public function getBudget()
	{
		return $this->budget;
	}

	/**
	 * @param int $budget
	 * @return Project
	 */
	public function setBudget($budget)
	{
		$this->budget = $budget;
	}
}
